elrond-wasm/src/types/managed/basic/big_int.rs

```diff
@@ -121,7 +145,7 @@ impl<M: ManagedTypeApi> BigInt<M> {
     pub fn zero() -> Self {
         let handle: M::BigIntHandle = M::static_var_api_impl().next_handle();
         // TODO: seting 0 will no longer be needed once we fix VM handle error
-        M::managed_type_impl().bi_set_int64(handle.clone(), 0);
+        Self::set_value(handle.clone(), 0);
```

Comment:
Since setting to 0 is always safe, I think we can just keep calling the low-level API. No need to do all the overflow checks for 0.

Reply:
Done.